src/moduls/mantenimientos/APIHandler.mantenimientos.js

```diff
@@ -0,0 +1,40 @@
+import APIHandler from '../../APIHandler';
+
+const url = 'https://apicrinmo.azurewebsites.net/maintenance/create';
+/* const url = 'http://localhost:4000/maintenance/create'; */
+
+const secondaryUrl = 'https://apicrinmo.azurewebsites.net/customer/create'
+/* const secondaryUrl = 'http://localhost:4000/customer/create' */
+
+
+const API_TOKEN = "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwibmFtZSI6ImxvY2FsLXRlc3QtY2xpZW50LTAyIiwiaWF0IjoxNzAyOTk1NzI3LCJleHAiOjE3MzQ1MzE3Mjd9.mSTrPCZ3uEKaiPYWcNuT5TsFEyRVd0foJMMVMjzczKA"
```

Comment:
```suggestion
const azureWebAppApiEndpointCreateCustomer_API_TOKEN = "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwibmFtZSI6ImxvY2FsLXRlc3QtY2xpZW50LTAyIiwiaWF0IjoxNzAyOTk1NzI3LCJleHAiOjE3MzQ1MzE3Mjd9.mSTrPCZ3uEKaiPYWcNuT5TsFEyRVd0foJMMVMjzczKA"
```
ESPERA WHAT?! que hace esto aquí? los tokens no se suben al repositorio!